### PR TITLE
[cosim] disable wave for ciRun

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

**Review by sequencer — DISMISSED** [review]

File: build.sc [hunk 1]
```diff
@@ -264,12 +266,11 @@ object tests extends Module {
          |
          |target_link_libraries(emulator PUBLIC $${CMAKE_THREAD_LIBS_INIT})
          |target_link_libraries(emulator PUBLIC libspike fmt::fmt glog::glog)  # note that libargs is header only, nothing to link
-         |target_compile_definitions(emulator PRIVATE COSIM_VERILATOR)
```
Comment (sequencer) [inline build.sc:267]: Why removing this

File: tests/emulator/src/dpi.cc [hunk 2]
```diff
@@ -1,6 +1,4 @@
-#ifdef COSIM_VERILATOR
 #include <VTestBench__Dpi.h>
-#endif
```
Comment (sequencer) [inline tests/emulator/src/dpi.cc:3]: Why

File: tests/emulator/src/dpi.cc [hunk 3]
```diff
@@ -1,6 +1,4 @@
...
```
Comment (sequencer) [inline tests/emulator/src/dpi.cc:2]: Keep it

**Review by sequencer — COMMENTED** [review]
don't remove `COSIM_VERILATOR`

File: build.sc (hunk 1 above)
Comment (ksco) [reply build.sc:267]: This seems a broken feature.

Comment (sequencer) [reply build.sc:267]: @SharzyL

**Review by sequencer — APPROVED** [review]

**Pull request merged**